x/auth/types/account_test.go

```diff
@@ -237,6 +238,15 @@ func TestModuleAccountJSONPB(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestBaseAccountJSONPB(t *testing.T) {
+	baseAccountJson := "{\"address\":\"link1rrfywnytlm87ywes0hvxn4rhm4grrn9qquqljc\",\"pub_key\":{\"type\":0,\"key\":null},\"account_number\":\"10\",\"sequence\":\"50\"}"
```

Comment:
You may want to use `` to make it simple.

```suggestion
	baseAccountJson := `{"address":"link1rrfywnytlm87ywes0hvxn4rhm4grrn9qquqljc","pub_key":{"type":0,"key":null},"account_number":"10","sequence":"50"}`
```